src/System Application/Test/MicrosoftGraph/src/MockHttpClientHandlerMulti.Codeunit.al

```diff
@@ -0,0 +1,85 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.Test.Integration.Graph;
+
+using System.RestClient;
+
+codeunit 135145 "Mock Http Client Handler Multi" implements "Http Client Handler"
+{
+    Access = Internal;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    var
+        _httpRequestMessages: List of [Text];
```

Comment:
I understand the `_` are added to make them Global, but it doesn't follow AL's styling. 
What we can do is keep `HttpRequestMessage` as global and in the local functions use `this.HttpRequestMessage := HttpRequestMessage`, same for the others.